solutions/tests/test_two_sum.py

```diff
@@ -0,0 +1,43 @@
+"""
+Unit tests for the two_sum function.
+"""
+
+import unittest
+from solutions.two_sum import two_sum
+
+
+class TestTwoSum(unittest.TestCase):
+    """
+    Unit tests for the two_sum function.
+    """
```

Comment:
Expand Test Coverage:
Add tests for edge cases:
Empty list ([]) or single-element list ([5]) to ensure proper handling or error raising.
```suggestion
    """
def test_empty_or_single_element_list(self):
    with self.assertRaises(ValueError):
        two_sum([], 9)
    with self.assertRaises(ValueError):
        two_sum([5], 5)
```